base_temporary_action/models/ir_actions_actions.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class IrActionsActions(models.Model):
+    _inherit = "ir.actions.actions"
+
+    is_temporary = fields.Boolean()
+
+    @api.autovacuum
+    def _gc_temporary_actions(self):
+        self.search(
+            [
+                ("is_temporary", "=", True),
+                (
+                    "create_date",
+                    "<",
+                    datetime.now() - relativedelta(hours=1),
```

Comment:
```suggestion
                    odoo.fields.Datetime.now() - relativedelta(hours=1),
```

To be explicit that we want the UTC datetime Odoo uses. In practice I think `datetime.datetime.now()` is patched by Odoo to return UTC, but I prefer the explicit version.